.github/dependabot.yml

```
@@ -23,3 +23,10 @@ updates:
     reviewers:
       - "bajtos"
       - "juliangruber"
+    - package-ecosystem: "docker"
+    directories:
+      - "/"
+    schedule:
+      interval: "daily"
+      time: "15:00"
```

```suggestion
      time: "09:00"
```

to match the other times configured